llvm/lib/Target/RISCV/RISCVVLOptimizer.cpp

```diff
@@ -257,6 +257,9 @@ static OperandInfo getOperandInfo(const MachineOperand &MO,
   // Vector Unit-Stride Instructions
   // Vector Strided Instructions
   /// Dest EEW encoded in the instruction and EMUL=(EEW/SEW)*LMUL
+  case RISCV::VLM_V:
+  case RISCV::VSM_V:
```

Comment:
No tests for VSM?

Reply:
Added, although I'm not sure that it makes sense to have an incompatible_eew test does it?